sdk/cosmos/src/operations/create_slug_attachment.rs

```diff
@@ -0,0 +1,128 @@
+use crate::headers::from_headers::*;
+use crate::prelude::*;
+use crate::resources::Attachment;
+use crate::{ResourceQuota, Slug};
+use azure_core::headers::{etag_from_headers, session_token_from_headers};
+use azure_core::prelude::*;
+use azure_core::SessionToken;
+use azure_core::{collect_pinned_stream, Request as HttpRequest, Response as HttpResponse};
+use bytes::Bytes;
+use chrono::{DateTime, Utc};
+
+#[derive(Debug, Clone)]
+pub struct CreateSlugAttachmentOptions<'a> {
+    content_type: Option<ContentType<'a>>,
+    if_match_condition: Option<IfMatchCondition<'a>>,
+    activity_id: Option<ActivityId<'a>>,
+    consistency_level: Option<ConsistencyLevel>,
+}
+
+impl<'a> CreateSlugAttachmentOptions<'a> {
+    pub fn new() -> Self {
+        Self {
+            content_type: None,
+            if_match_condition: None,
+            activity_id: None,
```

Comment:
I believe activity_id does not need to be set by the user. A pipeline policy will set this on the user's behalf.